src/prommis/uky/costing/ree_plant_capcost.py

```diff
@@ -3442,3 +3443,65 @@ def assert_Pyomo_object(b, name):
                     f"Argument {obj} of type {type(obj)} is not a supported object type. "
                     f"Ensure {name} is a Pyomo Param, Var, Expression, or ScalarExpression."
                 )
+
+    def economy_of_numbers(
+        blk, cum_num_units, cost_FOAK, CE_index_year, learning_rate=0.04
+    ):
+        """
+        Equations for Economy of Numbers (EoN) derived from NETL Quality Guidelines for Energy System Studies
```

Comment:
It looks like this QGESS is no longer publicly released (but i do see the references to it in other documents). Were you able to find a version of it?

Reply:
I just realized my version is missing from the official NETL page. However, the info is also in the review of the QGESS you shared earlier. Should I cite the review document or the QGESS report, considering that the review also cited the QGESS report?

Reply:
See my comment above.

If you cite the QGESS, check that your version is publicly available or reference a newer version with the same information. Your version might be on OSTI or another database.

Reply:
Unfortunately, it's also not on OSTI or any other database. Since it has the same information, I will use the CMU report. Thank you.

Reply:
Ok thank you! It looks like the reference is fixed below but I would also recommend removing the reference to QGESS here since it is no longer published. I also did not see that the CMU report published the equations, were they in the report or did i miss them?

Reply:
The CMU report also published the equations (Chapter 6, equations 2 and 3). However, the definition of variables in the report is slightly different from what I used in the code. For instance, they worked with a reduction in time as the cumulative production unit increased, and for ours, it's cost reduction.

Reply:
The equation formation used is shown in this publication: https://www.osti.gov/pages/biblio/1868500
I would recommend adding this as the citation for the equations.